src/parquet/column_chunk_reader.cc

```diff
@@ -0,0 +1,488 @@
+#include <seastar/parquet/column_chunk_reader.hh>
+#include "compression.hh"
+
```

Comment:
GitHub does not allow commenting on commit messages, so I'll do this here: before pushing upstream, big commits (e.g. 882025425aa5315aff81efc27a91daf3040e9b13) need to have a proper commit messages - e.g. a couple of sentences explaining what a Parquet reader is and why it was added. In principle, every commit should have a small commit message, except for the really trivial ones (e.g. fixing an obvious typo).